include/xtensor/xfixed.hpp

```diff
@@ -111,7 +112,8 @@ namespace xt
         template <std::size_t I, std::size_t... X>
         struct calculate_stride_row_major
         {
-            constexpr static std::ptrdiff_t value = at<sizeof...(X) - I, X...>::value * calculate_stride_row_major<I - 1, X...>::value;
+            constexpr static std::ptrdiff_t value = at<sizeof...(X) - I, X...>::value
+                                                    * calculate_stride_row_major<I - 1, X...>::value;
```

Comment:
See comment above:
```suggestion
            constexpr static std::ptrdiff_t value = 
                at<sizeof...(X) - I, X...>::value * calculate_stride_row_major<I - 1, X...>::value;
```
seems more logical